### PR TITLE
Add counters for dataset classes

### PR DESCRIPTION
**Pull request by jiangyukunok**
# What does this PR do?

This improves the logging in **examples/pytorch/text-classification/run_glue.py** by adding counts for different labels in train/validation/test datasets. The logs could be useful when working with classification tasks (spot class imbalance early, check data split consistency, etc).

test result:
04/20/2025 03:52:36 - INFO - __main__ - Class distribution in train set:
04/20/2025 03:52:36 - INFO - __main__ -   Label 1: 2474 (67.45%)
04/20/2025 03:52:36 - INFO - __main__ -   Label 0: 1194 (32.55%)
04/20/2025 03:52:36 - INFO - __main__ - Class distribution in validation set:
04/20/2025 03:52:36 - INFO - __main__ -   Label 1: 279 (68.38%)
04/20/2025 03:52:36 - INFO - __main__ -   Label 0: 129 (31.62%)
04/20/2025 03:52:36 - INFO - __main__ - Class distribution in test set:
04/20/2025 03:52:36 - INFO - __main__ -   Label 1: 1147 (66.49%)
04/20/2025 03:52:36 - INFO - __main__ -   Label 0: 578 (33.51%)

## Before submitting
- [x] This PR fixes a typo or improves the docs (you can dismiss the other checks if that's the case).
- [x] Did you read the [contributor guideline](https://github.com/huggingface/transformers/blob/main/CONTRIBUTING.md#create-a-pull-request),
      Pull Request section?
- [ ] Was this discussed/approved via a Github issue or the [forum](https://discuss.huggingface.co/)? Please add a link
      to it if that's the case.
- [ ] Did you make sure to update the documentation with your changes? Here are the
      [documentation guidelines](https://github.com/huggingface/transformers/tree/main/docs), and
      [here are tips on formatting docstrings](https://github.com/huggingface/transformers/tree/main/docs#writing-source-documentation).
- [ ] Did you write any new necessary tests?

## Who can review?
Anyone in the community is free to review the PR once the tests have passed. Feel free to tag
members/contributors who may be interested in your PR.

### CODE REVIEW

**Review by Rocketknight1 — APPROVED** [review]
Yes, this seems like a nice, clean addition!

**Comment by jiangyukunok** [comment]
Hi @ArthurZucker — just checking in to see if you had a chance to review this PR 🙏 
Matt already approved, let me know if any changes are needed!

**Comment by Rocketknight1** [comment]
This is simple enough that we don't need core maintainer approval, don't worry! Thank you for the PR
